manifests/simp_apache.pp

```diff
@@ -100,32 +100,22 @@
 # @copyright 2016 Onyx Point, Inc.
 #
 class simp_elasticsearch::simp_apache (
-  $manage_httpd                 = true,
-  $listen                       = '9200',
-  $proxyport                    = '9199',
-  $ssl_protocols                = ['+TLSv1','+TLSv1.1','+TLSv1.2'],
-  $ssl_cipher_suite             = hiera('openssl::cipher_suite',['HIGH']),
-  $ssl_certificate_file         = "/etc/pki/public/${::fqdn}.pub",
-  $ssl_certificate_key_file     = "/etc/pki/private/${::fqdn}.pem",
-  $ssl_ca_certificate_path      = '/etc/pki/cacerts',
-  $ssl_verify_client            = 'require',
-  $ssl_verify_depth             = '10',
-  $method_acl                   = {}
+  Boolean              $manage_httpd             = true,
+  Simplib::Port        $listen                   = 9200,
+  Simplib::Port        $proxyport                = 9199,
+  Array[String]        $ssl_protocols            = ['+TLSv1','+TLSv1.1','+TLSv1.2'],
+  Array[String]        $ssl_cipher_suite         = simplib::lookup('options::openssl::cipher_suite', { 'default_value' => ['HIGH'] },
+  Stdlib::AbsolutePath $ssl_certificate_file     = "/etc/pki/public/${facts['fqdn']}.pub",
```

Comment:
A few thoughts about the ssl stuff now that it's been migrated to class parameters:
- We are using a class parameter naming scheme for anything ssl related, outlined in the doc.
- This is a weird situation because the old behavior was to use the same certs as simp_apache, but that's not the best way to manage them IMO.  We may consider using the pki::source catalyst with a pki::copy like everything else.
- If possible, pki should be managed via the pki catalyst.